src/assets/translations.data.json

```diff
@@ -290,6 +290,9 @@
     "preferences.disableTeleporter": "Disable teleporter",
     "preferences.movementSpeedModifier": "Movement speed modifier",
     "preferences.disableAutoPixelRatio": "Disable automatic pixel ratio adjustments",
+    "preferences.disableEchoCancellation": "Disable microphone echo cancellation (requires restart)",
```

Comment:
If these are only available on FxR, perhaps add that into the description or put it into a separate heading (like the touchscreen preferences)

Reply:
These are available on any platform. We only will set them to true (disable those features) in the case of FxR.

Reply:
I only see them being used in a `isFirefoxReality` block. Am I missing something?

Reply:
Oh you're right, im sorry. I need to be reading these preferences where i do: 
```
    constraints.audio.echoCancellation = true;
    constraints.audio.noiseSuppression = true;
    constraints.audio.autoGainControl = true;
```